source/uwp/Renderer/lib/TileControl.cpp

```diff
@@ -55,13 +55,13 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
 
     winrt::Windows::Foundation::Size TileControl::MeasureOverride(winrt::Windows::Foundation::Size const& availableSize)
     {
-        return TileControl_base::MeasureOverride(availableSize);
+        return __super::MeasureOverride(availableSize);
```

Comment:
Please go back to `_base`. `__super` is an MSVC nonstandard extension. We should be moving away from nonstandard code, not running toward it.